src/backends/backend_interface.py

```diff
@@ -107,3 +107,10 @@ def warnet_from_deployment(self, warnet):
         Rebuild a warnet object from an active deployment
         """
         raise NotImplementedError("This method should be overridden by child class")
+
+    @abstractmethod
+    def fetch_ip_addresses(self, tank) -> bool:
```

Comment:
in "backend: remove ipv4 from compose backend" (https://github.com/bitcoin-dev-project/warnet/pull/254/commits/50107b7f1dd88a1f0d393ed0e64b2ef8b19fddc7)

nit: shouldn't this be `fetch_ip_address`, since it only operates on a single tank? also not sure if this matters in python if the abstractmethod return type differs from the implementation, but if we're returning a bool, maybe we should call this `set_ip_address`?

Reply:
Agree